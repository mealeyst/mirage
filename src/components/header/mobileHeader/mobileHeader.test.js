import React from 'react'
import 'jest-styled-components'

import { MobileHeader } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) MobileHeader', () => {
  const createMobileHeader = (props) => {
    return shallowWithTheme(<MobileHeader {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createMobileHeader())
    .toMatchSnapshot()
  })
})
