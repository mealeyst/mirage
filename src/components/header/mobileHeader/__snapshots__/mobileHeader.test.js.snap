// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) MobileHeader matching the snapshot 1`] = `
.c0 {
  min-height: 4.8rem;
  background-color: #FFFFFF;
  box-shadow: 0 0.2rem 0.1rem 0 rgba(0,0,0,.16);
  top: 0;
  left: 0;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0 0.5rem 0 1rem;
  box-sizing: border-box;
}

<header
  className="c0"
/>
`;
